### PR TITLE
Add code_merge.py

### PR DESCRIPTION
**Pull request by as2388**
Can be reviewed independently of #239 but I don't recommend using it without merging #239 first.

Automatically assigns new labels with the id of the merged code when doing a code_merge in a code scheme.

### CODE REVIEW

File: data_tools/code_merge.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+from core_data_modules.traced_data import Metadata
+from core_data_modules.util import TimeUtils
+
+Logger.set_project_name("CodeMerge")
+log = Logger(__name__)
+
+parser = argparse.ArgumentParser(description="Performs a code merge on a local dataset. "
+                                             "To use with Coda, use get.py, code_merge.py, then set.py")
+parser.add_argument("messages_input_file_path", metavar="messages-input-file-path",
+                    help="Path to the Coda messages file to perform the code merge on")
```
Comment (IsaackMwenda) [inline data_tools/code_merge.py]:
```suggestion
                    help=" Path to the Coda messages file to read the messages -> to be merged codes from")
```

File: data_tools/code_merge.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+parser.add_argument("source_code_ids", metavar="source-code-ids", nargs="+",
```
[the remark this answers is not in the document]
Comment (as2388) [reply data_tools/code_merge.py]: This is much clearer, thanks. Changed.

[the remark this answers is not in the document]
Comment (as2388) [reply data_tools/code_merge.py]: This is much clearer, thanks. Changed.

File: data_tools/code_merge.py (hunk 1 above)
Comment (as2388) [reply data_tools/code_merge.py]: Updated to "Path to the file to read the Coda messages to be code-merged from"

**Pull request merged**